weave-js/src/components/Panel2/ChildPanelExportReport/ReportSelection.tsx

```diff
@@ -74,9 +74,9 @@ export const ReportSelection = ({
       } as any);
     }),
   });
-  const reports = useNodeValue(reportsMetaNode ?? w.voidNode(), {
-    skip: entities.loading,
-  });
+  const reports = useValue(reportsMetaNode ?? w.voidNode());
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useEffect(() => reports.refresh(), []);
```

Comment:
it wants `reports` to be in the deps array, but doing so causes infinite refreshing. presumably `useValue` should be memoizing its result but that doesn't seem to be working properly. I decided to take the lazy way out and disable the lint check instead of dig into `useValue`. the empty array means we only refresh on mount which totally fulfills what I want anyway